### ISSUE TITLE
Location comparison dialog: Load Similar locations does not load for states

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#413]
When clicking "Load Similar" in the location comparison dialog when comparing states, the Loading button loads indefinitely.

Steps to reproduce:
1) Go to side nav > Location filter > California & Nevada
2) Click Compare Locations
3) In the dialog, click Load Similar
4) See the loading button load indefinitely

Expected results: up to 5 additional states load in the table
Actual results: no additional states load, button says "loading" indefinitely

See screenshot: 
![Load Similar - states](https://user-images.githubusercontent.com/76966798/104386753-9b543680-54ea-11eb-9584-ef73942e2cdb.png)

**Comment by Lane** [comment]
waiting on client deliverable for similar states before this can be fixed

**Comment by laurendoone** [comment]
@Lane is this part of the final data set? Or a separate deliverable I should follow up with them on?

**Comment by Lane** [comment]
@laurendoone part of the final data set!

**Comment by Lane** [comment]
no longer relevant, states will not have similar places.  replaced by #480 0
